packages/shared/src/styles/components/buttons.css

```
@@ -229,10 +230,12 @@
   &[aria-pressed="true"] {
     --button-background: var(--button-pressed-background);
     --button-icon: var(--button-hover-icon);
+    transition: all 0.2s ease-in-out;
   }
 
   &:hover, &.hover, &:focus:focus-visible {
     --button-icon: var(--button-hover-icon);
+    transition: all 0.2s ease-in-out;
```

Same goes here, just tagging for visibility 🚢 
```suggestion
    transition: background-color 0.2s ease-in-out;
```